### ISSUE TITLE
Why limit it to music?

### ISSUE THREAD

**Issue by barnacs** [Musicoin/discussion#2]
From the draft it seems to me that the same method could be applied to most forms of digital creative content - videos, photos, images, books, articles, models, software, etc.
Is there any reason you opted to limit the use-case to music only?

**Comment by immartian** [comment]
Good point, it's not necessary to restrict the model to be used only for Music, and it's the ultimate reason we just want to start and share the design.  However, for music industry, there's chronicle disease that musicians can't get paid fairly, the same time, ton of millions of profit goes into labels pocket.  To solve this 15 Billion dollar issue, can be a great illustration to solve others.
